conf/application.conf

```diff
@@ -44,6 +44,7 @@ play.modules.enabled += "uk.gov.hmrc.play.audit.AuditModule"
 play.modules.enabled += "uk.gov.hmrc.play.bootstrap.AuthModule"
 play.modules.enabled += "uk.gov.hmrc.play.bootstrap.graphite.GraphiteMetricsModule"
 play.modules.enabled += "config.Module"
+play.modules.enabled += "config.TransitionModule"
```

Comment:
This needs to be
```
play.additional.module = "config.TransitionModule"
play.modules.enabled += ${play.additional.module}
```
otherwise we can't override it. Can also remove `play.modules.enabled += "config.Module"`